SUPPORTED_APIS.md

```diff
@@ -25,7 +25,7 @@ The `status` options are:
 | `DEBUG` | `debug_traceBlockByHash` | `NOT IMPLEMENTED`<br />[GitHub Issue #63](https://github.com/matter-labs/era-test-node/issues/63) | Returns structured traces for operations within the block of the specified block hash |
 | `DEBUG` | `debug_traceBlockByNumber` | `NOT IMPLEMENTED`<br />[GitHub Issue #64](https://github.com/matter-labs/era-test-node/issues/64) | Returns structured traces for operations within the block of the specified block number |
 | `DEBUG` | `debug_traceTransaction` | `NOT IMPLEMENTED`<br />[GitHub Issue #65](https://github.com/matter-labs/era-test-node/issues/65) | Returns a structured trace of the execution of the specified transaction |
-| `ETH` | `eth_accounts` | `NOT IMPLEMENTED`<br />[GitHub Issue #50](https://github.com/matter-labs/era-test-node/issues/50) | Returns a list of addresses owned by client |
+| `ETH` | `eth_accounts` | `SUPPORTED` | Returns a list of addresses owned by client |
```

Comment:
Should we add a note that this is hard-coded to return with the Rich Accounts?

Reply:
Yeah, I suppose we should indicate PARTIALLY instead of supported? @MexicanAce

Reply:
From what I can tell, most other nodes just return an empty array... and reading the documentation of this endpoint, it makes sense to return the rich accounts.... So maybe we don't need the note that it's hard-coded after all? 🤔